h/views/api_groups.py

```diff
@@ -15,6 +15,8 @@
 def groups(request):
     authority = request.params.get('authority')
     document_uri = request.params.get('document_uri')
+    expand = request.GET.getall('expand') or []
```

Comment:
Given that pyramid does some sanitization for us and invalid `expand` values are ignored, I didn't put any validation here. Is that OK?

Reply:
This seems fine to me